root/statistics/MusicBrainzHistory.js

```diff
@@ -0,0 +1,51 @@
+/*
+ * @flow strict-local
+ * Copyright (C) 2021 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import Layout from '../layout';
+import expand2react from '../static/scripts/common/i18n/expand2react';
+
+type PropsT = {
+  +events: Array<StatisticsEventT>,
+};
+
+const MusicBrainzHistory = ({
+  events,
+}: PropsT): React.Element<typeof Layout> => {
+  const sortedEvents = events.reverse();
```

Comment:
Can the Perl send them in the correct order?

Reply:
Sure.